resteasy/src/test/resources/com/cerner/beadledom/resteasy/expected/api-docs-health.json

```diff
@@ -1,327 +0,0 @@
-{
```

Comment:
Was this not being used in a test?

Reply:
The test it was in I removed from `com/cerner/beadledom/resteasy/ResteasyServiceSpec.scala`. Pretty sure it was because the endpoint it was testing no longer exists.

Reply:
Is that because swagger2 no longer has the separate endpoints and groups it all together?

Reply:
As far as I have seen from their docs, it is just the root `/api-docs` endpoint.